jobs/ThoughtLeadr-Python-Engineer-NativeKit.html

```diff
@@ -0,0 +1,54 @@
+---
+title: Python Engineer
+company: ThoughtLeadr
+url: http://www.thoughtleadr.com/
+location: Austin, Texas
+contract: permanent
+contact:
+    name: Todd Cullen
+    email: jobs@thoughtleadr.com
+    phone: (415) 658-5744
+created: !!timestamp '2017-02-08'
+tags:
+  - Austin
+  - Texas
+  - USA
+  - Python
+  - Flask
+  - React
+  - Javascript
+---
+
+# About the Job
+
+At ThoughtLeadr, we provide our clients with best-in-class analytics and advertising solutions. The Engineering and Products teams work tightly together to create a shared vision for the future of the company. We take pride in the craftsmanship of our code and the reliability of our systems.
+
+We are looking for a Python Engineer to help us develop the consumer facing part of our flagship product, NativeKit. You should be passionate about growing as an engineer, learning different technologies, and solving complex problems that impact millions of users.
+
+# Responsibilities
+
+Develop, deploy, and maintain robust and high performance production systems and features
+Establish expertise in designing, analyzing and troubleshooting large-scale distributed systems
+Partner with internal teams to define and implement solutions that improve internal business processes
+Systematic problem solving approach, coupled with a strong sense of ownership and drive
+
+# Qualifications
+
+## Minimum qualifications:
+
+* BS degree in Computer Science or related field, or equivalent practical experience.
+* 2 years of hands-on experience coding database-driven web applications in Python and Javascript
```

Comment:
We generally advise against giving a specific number of years experience. Giving years might discriminate against younger or older candidates.